draft-ietf-mls-protocol.md

```diff
@@ -4854,45 +4854,63 @@ outside the group:
 * Group ID
 * Group membership
 * Group extensions
-* Frequency of epoch changes
+* Current epoch and frequency of epoch changes
 * Frequency of application messages within an epoch
 
+### DS Architecture
+
+The following is relevant for applications where the DS is an entity less
+trusted than the members’ clients (e.g. a serverside component).
+
 This information is typically protected from parties other than the DS by
 applying "hop by hop" transport encryption on communications with the DS (in
 contrast to the "end to end" protections provided by MLS). The protection this
 information receives with regard to the DS depends on how MLS is operated.
 Overall, MLS does not provide robust protections against the DS observing the
 group's metadata.
 
+With the exception of Group ID and Epoch, all of the following is only relevant
+when Commits and Proposal messages use the MLSPlaintext framing. If instead
+MLSCiphertext is used, the DS cannot observe the content of the messages
+directly.
+
+### Group ID and Epoch
+
 MLS provides no mechanism to protect the group ID and epoch of a message from
 the DS, so group ID and the frequency of messages and epoch changes are not
 protected against inspection by the DS.
 
+### Extensions
+
+Group extensions are set via GroupContextExtensions proposals and thus exposed
+to the DS. Extensions can also be exposed via {{groupinfo-objects}}.
```

Comment:
```suggestion
to the DS. Extensions can also be exposed via GroupInfo objects (see {{groupinfo-objects}}).
```

Otherwise it will render as "exposed via Section X.X.X"

This will render as "exposed via Section X.X.X".